### PR TITLE
[ES-132] Add find total balance

### PR DESCRIPTION
**Pull request by jeddie-yhc**

### CODE REVIEW

File: store/service_store.go [hunk 1]
```diff
@@ -59,6 +59,9 @@ type ServiceManager interface {
 	// Noted that the return block number may be different from the input one because
 	// we don't have state in the input one.
 	GetERC20Balance(ctx context.Context, contractAddress, address common.Address, blockNr int64) (*decimal.Decimal, *big.Int, error)
+
+	// Subscriptions store
+	FindTotalBalance(blockNumber int64, token common.Address, group int64) (result *model.TotalBalance, err error)
```
Comment (kimmylin) [inline store/service_store.go]: do M&M want to query all tokens together or one by one?
if former, you should pass a list of tokens and do everything in one sql stmt with `IN` clause.

Comment (tailingchen) [reply store/service_store.go]: I think subscription store can support query many tokens even if M&M would like to query one by one.

Comment (jeddie-yhc) [reply store/service_store.go]: According to the previous discussion with M&M, they want to query one by one as the first priority for their existed logic (please refer to ES-132.) The second priority is to query all tokens once API then.

**Comment by jeddie-yhc** [comment]
Thanks @markya0616! I've update the last commit message in https://github.com/getamis/eth-indexer/pull/24/commits/133be462b67709cb31b2aaaf9775c5237c50a369.

**Pull request merged**